### PR TITLE
[RHPAM-2796] Test for aborted type task when aborting process

### PR DESCRIPTION
**Pull request by gmunozfe**

### CODE REVIEW

File: jbpm-audit/src/test/java/org/jbpm/process/audit/StandaloneAuditLogServiceTest.java [hunk 1]
```diff
@@ -106,4 +108,23 @@ public void setAuditLogServiceForCommandTest() {
...
+        long processInstanceId = ksession.startProcess(PROCESS_ID).getId();
+        
+        logger.info("Checking process instances for process '{}' before aborting", PROCESS_ID);
+       
+        ksession.abortProcessInstance(processInstanceId);
+        
+        List<NodeInstanceLog> nodeInstances = auditLogService.findNodeInstances(processInstanceId);
+        
+        assertEquals("There should be 4 node instances", 4, nodeInstances.size());
+        assertEquals("Active task node instance when aborting process should be at ABORTED type -2-",
```
[the remark this answers is not in the document]
Comment (gmunozfe) [reply jbpm-audit/src/test/java/org/jbpm/process/audit/StandaloneAuditLogServiceTest.java:125]: Done

File: jbpm-audit/src/test/java/org/jbpm/process/audit/StandaloneAuditLogServiceTest.java [hunk 2]
```diff
@@ -59,7 +60,8 @@ public void setUp() throws Exception {
         ksession = createKieSession(kbase, env);
         new JPAWorkingMemoryDbLogger(ksession);
         ksession.getWorkItemManager().registerWorkItemHandler("Human Task", new SystemOutWorkItemHandler());
-       
+        //Register a workitem that does not complete it when executed
+        ksession.getWorkItemManager().registerWorkItemHandler("No completed task", new DoNothingWorkItemHandler());
```
[the remark this answers is not in the document]
Comment (gmunozfe) [reply jbpm-audit/src/test/java/org/jbpm/process/audit/StandaloneAuditLogServiceTest.java:64]: Done

File: jbpm-audit/src/test/resources/ruleflow4.rf [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+         xs:schemaLocation="http://drools.org/drools-5.0/process drools-processes-5.0.xsd"
+         type="RuleFlow" name="ruleflow4" id="com.sample.ruleflow4" package-name="org.drools.core.process.workflow" >
+
+  <header>
+  </header>
+
+  <nodes>
+    <start id="1" name="Start" />
+    <workItem id="2" name="Task" >
+      <work name="No completed task" >
```
[the remark this answers is not in the document]
Comment (gmunozfe) [reply jbpm-audit/src/test/resources/ruleflow4.rf:13]: Done

File: jbpm-audit/src/test/java/org/jbpm/process/audit/StandaloneAuditLogServiceTest.java [hunk 4]
```diff
@@ -106,4 +108,23 @@ public void setAuditLogServiceForCommandTest() {
         processInstances = setAuditLogServiceAndExecute(new FindProcessInstancesCommand(PROCESS_ID));
         assertEquals(0, processInstances.size());
     }
+    
+    @Test
+    public void whenAbortingProcess_activeTaskShouldBeAtAbortedType() {
```
[the remark this answers is not in the document]
Comment (gmunozfe) [reply jbpm-audit/src/test/java/org/jbpm/process/audit/StandaloneAuditLogServiceTest.java]: Done

**Comment by gmunozfe** [comment]
Thanks a lot for your review @afalhambra ! All the items have been resolved.

**Comment by mareknovotny** [comment]
can you approve this too @afalhambra ;)

**Pull request merged**